openff/toolkit/utils/utils.py

```diff
@@ -641,9 +639,9 @@ def detach_units(unit_bearing_dict, output_units=None):
     return unitless_dict, unit_dict
 
 
-def serialize_numpy(np_array):
+def serialize_numpy(np_array) -> Tuple[bytes, Tuple[int]]:
     """
-    Serializes a numpy array into a JSON-compatible string. Leverages the numpy.save function,
+    Serializes a numpy array into a JSON-compatible big-endian bytestring. Leverages the numpy.save function,
     thereby preserving the shape of the input array
 
     from https://stackoverflow.com/questions/30698004/how-can-i-serialize-a-numpy-array-while-preserving-matrix-dimensions#30699208
```

Comment:
```suggestion
    Serializes a numpy array into a JSON-compatible big-endian bytestring.
```

Non-blocking -- perhaps I'm missing something, but this docstring no longer seems accurate; np.save isn't really used?

Reply:
Correct, this either didn't ever use `np.save` or did so in a way that was lost to the sands of time/git history